test/functional/wallet_dump.py

```diff
@@ -87,12 +86,6 @@ def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
     def setup_network(self):
-        # TODO remove this when usehd=1 becomes the default
-        # use our own cache and -usehd=1 as extra arg as the default cache is run with -usehd=0
-        self.options.tmpdir = os.path.join(self.options.tmpdir, 'hd')
-        self.options.cachedir = os.path.join(self.options.cachedir, 'hd')
-        self._initialize_chain(extra_args=self.extra_args[0])
-        self.set_cache_mocktime()
```

Comment:
15415 why are these changes included

Reply:
This was a workaround for the hd/non-hd wallets conflict (cache is created with non-hd). 15415 fixed it, so this hack is no longer need.